tests/core/dom/range/getclientrects.js

```diff
@@ -0,0 +1,49 @@
+/* bender-tags: editor,dom,range */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
```

Comment:
You don't really need to use editor here. Actually you can work on regular HTML elements. Just go with:

```js
var rng = new CKEDITOR.dom.range( CKEDITOR.document );
```

and play with your range.